go/libkb/globals.go

```diff
@@ -176,14 +176,21 @@ func (g *GlobalContext) ConfigureKeyring() error {
 	return nil
 }
 
-func VersionMessage(linefn func(string)) {
+type GitHashGetter interface {
+	Hash() string
+}
+
+func (g *GlobalContext) VersionMessage(linefn func(string)) {
 	linefn(fmt.Sprintf("Keybase CLI %s", VersionString()))
 	linefn(fmt.Sprintf("- Built with %s", runtime.Version()))
+	if h, ok := interface{}(g).(GitHashGetter); ok {
```

Comment:
It seemed worthwhile to still be able to build without doing `go generate` in the keybase directory - though doing so means less and less, the more generated stuff we add (icon and version resources, etc.).

Backing up a little, notice that this change will make "go generate" in the keybase directory generate libkb/hash_windows.go, and the client can build fine with or without doing that. I was leery of introducing a hard requirement.